contrib/node/src/python/pants/contrib/node/subsystems/node_distribution.py

```diff
@@ -45,6 +48,13 @@ def register_options(cls, register):
                  NodeDistribution.VALID_PACKAGE_MANAGER_LIST.keys()))
       register('--yarnpkg-version', advanced=True, default='v0.19.1', fingerprint=True,
                help='Yarnpkg version. Used for binary utils')
+      register('--eslint-setupdir', advanced=True, fingerprint=True,
+               help='Find the package.json under this dir for installing eslint and plugins.')
```

Comment:
Should this say something about committing the `yarn.lock` file in this directory, or something?